galleries.py

```
@@ -512,376 +534,177 @@ def _gallery(fileroot, product_ids, catalog, links,
-        # Locate target of "<"
-        if k > 0:
-            neighbors[1] = sublist[k-1]
-        elif j == 0:
-            neighbors[1] = None
-        elif sublist_is_open[j-1]:
-            neighbors[1] = links[j-1][-1]
-        else:
-            neighbors[1] = links[j-1][0]
+        return data
+    
+    def create_previous_next_menu(isSublist, previous_link):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `create_previous_next_menu`

(ANN202)